saltgui/static/scripts/RunType.js

```diff
@@ -43,18 +43,25 @@ export class RunType {
   }
 
   static setRunTypeDefault () {
-    RunType.menuRunType._value = "normal";
+    // Retrieve last used runType
+    let runType = Utils.getStorageItem("local", "runtype");
+    // Set default if previous runtype not stored
+    if (runType !== "normal" && runType !== "async") {
+      runType = "normal";
+    }
     RunType._updateRunTypeText();
     // reset the title to the absolute minimum
     // so that the menu does not stand out in trivial situations
     RunType.menuRunType.setTitle("");
   }
 
   static getRunType () {
-    const runType = RunType.menuRunType._value;
+    let runType = RunType.menuRunType._value;
     if (runType === undefined || runType === "") {
-      return "normal";
+      runType = "normal";
     }
+    // Store last used runType
+    Utils.setStorageItem("local", "runtype", runType);
     return runType;
```

Comment:
when a page-refresh is done, all GUI values are unavoidably lost.
this function would then reset the value to "normal" and also save it.
- saving the value should only be done in function _updateRunTypeText, as that is the callback function from the dropdownlist.
- instead of `runType = "normal"`, the saved value should be retrieved: `runType = Utils.getStorageItem("local", "runtype", "normal");` (which itself will have a default of 'normal')

Reply:
Moved to ` _updateRunTypeText,` and suggested modification for retrieval applied, all tested and linted.